### DISCUSSION TITLE
사용자 일괄 등록 할 때 학교를 지정해서 등록하는데 모든 학교가 등록되는 현상이 있어요.

### DISCUSSION THREAD

**Discussion by KUBONKIL** [bmrdevteam/Altsis#282] [질문과 답변]
![image](https://github.com/bmrdevteam/school-information-system/assets/143776178/7900b1a6-b711-448c-b21d-8bcf53363354)
저는 사용자 등록할 때 tabschool로 지정 했는데 모든 학교가 등록이 됩니다.

**Comment by KUBONKIL** [comment]
관리자-사용자관리-사용자 생성-사용자 일괄 생성-3.학교선택 입니다.

**Comment by jessie129j** [comment]
해당 버그가 수정되었습니다([v1.9.12](https://github.com/bmrdevteam/school-information-system/releases/tag/v1.9.12))

**Reply by KUBONKIL** [reply]
버그가 수정된 것이 확인되었습니다. 고맙습니다. 고생하셨습니다.
